### PR TITLE
Update cisco kne topology

### PR DESCRIPTION
**Pull request by mojiiba**
This pull updates the kne topology by adding services, updating the interface name and constraint. This can be used to run otg tests.

### CODE REVIEW

**Comment by alexmasi** [comment]
Once https://github.com/openconfig/kne/pull/303 is tested and merged, those changes should be synced here and I will approve. Thanks

File: topologies/kne/cisco_ixia.textproto [hunk 1]
```diff
@@ -32,13 +32,8 @@ nodes: {
     os: "ios-xr"
     config: {
         file: "cisco.config"
-        image: "8000e:latest"
```
Comment (greg-dennis) [inline topologies/kne/cisco_ixia.textproto:35]: Why was the image removed? It's convenient for us to always have an image specified, even if that means we rewrite it on import.

Comment (mojiiba) [reply topologies/kne/cisco_ixia.textproto:35]: I did it based on the @alexmasi  comment on the kne pr, I am ok with reverting the change if u both agree.

Comment (alexmasi) [reply topologies/kne/cisco_ixia.textproto:35]: Yes Moji, for this repo we prefer to leave the image string explicitly in the proto (but thats not the case for the kne repo examples)

Sorry about any confusion there

Comment (mojiiba) [reply topologies/kne/cisco_ixia.textproto:35]: sure, done

**Review by alexmasi — APPROVED** [review]

**Review by thesrinath — APPROVED** [review]

**Pull request merged**